dev-build/conf/routes

```diff
@@ -4,7 +4,7 @@
 
 GET         /humans.txt                                                                                              dev.DevAssetsController.humans()
 GET         /surveys/*file                                                                                           dev.DevAssetsController.surveys(file)
-GET         /commercial/test-page                                                                                    controllers.commercial.CreativeTestPage.allComponents
+GET         /commercial/test-page                                                                                    controllers.commercial.CreativeTestPage.allComponents(k: List[String])
```

Comment:
Someone did it on the commercial app but not in dev-build ...

It basically looks for a k param in the query string. So if you go ?k[]=lol&k[]=dongs it'd pass List("lol", "dongs")